README.md

```diff
@@ -1,4 +1,7 @@
 # Something Borrowed
+#### Looking for a way to cut down on your expenses? Look no further than our website! 
+#### Something Borrowed allows you to lend and borrow household items within your neighbourhood. Whether you need a lawn mower for a one-time use or a set of dishes for a dinner party, Something Borrowed has you covered. 
+#### With our website, you can save money and get the items you need without having to buy them yourself. So why not give us a try? You won't be disappointed!
```

Comment:
For the third line, could we explain the app a bit more? I think this sells the app but it a bit light on the functionality. Maybe about how you can search for items based on category, location, or rating. Or mention that users have the ability to make an account, temporarily make items available if they need to use them themselves, or they can view a history of their borrowed items.
We could touch on what types of categories in the second second as well.

Reply:
ooh ok, sounds good, will include that